src/app/query/bitcoin/ordinals/brc20/brc20-tokens.query.ts

```diff
@@ -65,3 +76,78 @@ export function useBrc20TokensByAddressQuery<T extends unknown = FetchBrc20Token
     ...options,
   });
 }
+export function useBrc20TokensQuery() {
+  const network = useCurrentNetwork();
+  const keychain = useTaprootCurrentAccountPrivateKeychain();
```

Comment:
Would be better to do this from the signer hook and avoid `getTaprootAddress` if possible.

Think should work:

```ts
const createSigner = useCurrentAccountTaprootSigner();
const address = createSigner(index).address;
```